service/src/main/java/bio/terra/landingzone/service/landingzone/azure/model/LandingZoneResourcesByPurpose.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.landingzone.service.landingzone.azure.model;
+
+import java.util.List;
+import java.util.Map;
+
+public record LandingZoneResourcesByPurpose(
+    Map<String, List<LandingZoneResource>> generalResources,
```

Comment:
Mentioned above, but wondering if this should use the `ResourcePurpose` and `SubnetResourcePurpose` types. (They both extend from `ExpandableStringEnum` and have equals/hashCode.)

Reply:
Fixed this